resources/definitions/zonestar_Z9.def.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "ZONESTAR Z9",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "ZONESTAR",
+        "manufacturer": "ZONESTAR",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0],        
+        "machine_extruder_trains":
+        {
+            "0": "zonestar_mixvtool_0",
+            "1": "zonestar_mixvtool_1",
+            "2": "zonestar_mixvtool_2",
+            "3": "zonestar_mixvtool_3",
+            "4": "zonestar_mixvtool_4",
+            "5": "zonestar_mixvtool_5",
+            "6": "zonestar_mixvtool_6",
+            "7": "zonestar_mixvtool_7"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "ZONESTAR Z9" },
+        "layer_height": { "default_value": 0.2 },
+        "machine_extruder_count": { "default_value": 8 },
+        "machine_heated_bed": { "default_value": true },
+        "machine_center_is_zero": { "default_value": false },
+        "machine_height": { "default_value": 400 },
+        "machine_gcode_flavor": { "default_value": "RepRap (Marlin/Sprinter)" },
+        "machine_depth": { "default_value": 310 },
+        "machine_width": { "default_value": 310 },

```

Comment:
According to [this sales page](https://www.electroca.com/product/zonestr-z9-large-size-aluminum-frame-dual-extruder-mixing-color/) the printer's build volume is 300x300x400mm. Did you make a mistake here, or is the sales page incorrect?

Reply:
yes, please keep 310x310x400mm, we set a little small than the real size on sales page